### PR TITLE
feat: add task to upload sourcemaps to sentry

### PR DESCRIPTION
**Pull request by y-lohse**
cc the banksters team since it might be of interest to you as well @kosssi @ptbrowne @drazik @doubleface

### CODE REVIEW

**Comment by y-lohse** [comment]
[(1,5j) 📱 Améliorer notre gestion de Sentry](https://trello.com/c/oaQKEI6x/811-15j-%F0%9F%93%B1-am%C3%A9liorer-notre-gestion-de-sentry)

File: config/webpack.config.prod.js [hunk 1]
```diff
@@ -6,10 +6,11 @@ module.exports = {
   output: {
     filename: '[name].[hash].min.js'
   },
-  devtool: false,
+  devtool: '#source-map',
```
Comment (goldoraf) [inline config/webpack.config.prod.js]: Why the `#`? Are you sure it is necessary?

Comment (y-lohse) [reply config/webpack.config.prod.js]: That's actually a good question, I just copied it from the `dev` config. I'll experiment a bit more [with that](https://webpack.js.org/configuration/devtool/), looks like there are better options.

Comment (y-lohse) [reply config/webpack.config.prod.js]: You were correct, there's no need for the `#`. I also switched the dev sourcemaps to a faster version.

**Pull request merged**